src/Microsoft.NetCore.Analyzers/Core/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1179,4 +1179,19 @@
   <data name="MaybeDisableHttpClientCRLCheckMessage" xml:space="preserve">
     <value>HttpClient may be created without enabling CheckCertificateRevocationList</value>
   </data>
+  <data name="DefinitelyUseCreateEncryptorWithNonDefaultIV" xml:space="preserve">
```

Comment:
This name confuses me. It contradicts what the message is saying. Should it be `DontUseCreateEncryptorWithNonDefaultIV`?

Reply:
There're two cases. One is that we are 100% sure that CreateEncryptor uses non-default IV. The other is that non-default IV can be used but we are not sure. For example in the case below, non-default IV will be used when it fails to meets the requirement r.Next(6) == 4:
        var aesCng  = new AesCng();
        Random r = new Random();

        if (r.Next(6) == 4)
        {
            aesCng.IV = rgbIV;
        }

        aesCng.CreateEncryptor();

So, we created these two rules (Definitely and Maybe) for different scenarios.